src/AvaloniaEdit/Editing/TextArea.cs

```diff
@@ -1049,13 +1049,25 @@ public IControl GetControlInDirection(NavigationDirection direction, IControl fr
 
         public IList<RoutedCommandBinding> CommandBindings { get; } = new List<RoutedCommandBinding>();
 
-        public bool IsLogicalScrollEnabled => true;
+        public bool IsLogicalScrollEnabled { get; } = true;
```

Comment:
Oh, not what I meant. The expression syntax is preferred, so undo these changes.

I meant that all of the `ILogicalScrollable` members should not be public. So instead of `public bool IsLogicalScrollEnabled => true`, use `bool ILogicalScrollable.IsLogicalScrollEnabled => true`